mllib/src/main/scala/org/apache/spark/ml/clustering/BisectingKMeans.scala

```diff
@@ -302,6 +307,8 @@ object BisectingKMeans extends DefaultParamsReadable[BisectingKMeans] {
  * @param featuresCol  Name for column of features in `predictions`.
  * @param k  Number of clusters.
  * @param numIter  Number of iterations.
+ * @param trainingCost Sum of squared distances to the nearest centroid for all points in the
```

Comment:
Would the cost ever be something besides sum of squares? maybe not, just wondering if we should say here what the cost function is